src/macaron/slsa_analyzer/pypi_heuristics/metadata/empty_project_link.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""Analyzer checks there is no project link of the package."""
+
+from macaron.slsa_analyzer.package_registry.pypi_registry import PyPIApiClient
+from macaron.slsa_analyzer.pypi_heuristics.analysis_result import RESULT
+from macaron.slsa_analyzer.pypi_heuristics.base_analyzer import BaseAnalyzer
+from macaron.slsa_analyzer.pypi_heuristics.heuristics import HEURISTIC
+
+
+class EmptyProjectLinkAnalyzer(BaseAnalyzer):
+    """Analyzer checks heuristic."""
+
+    def __init__(self, api_client: PyPIApiClient) -> None:
+        super().__init__(name="empty_project_link_analyzer", heuristic=HEURISTIC.EMPTY_PROJECT_LINK)
+        self.api_client = api_client
+
+    def _get_links_total(self) -> tuple[int, dict] | None:
+        """Get total number of links.
+
+        Returns
+        -------
+            int | None: Total number of links.
+        """
+        project_links: dict | None = self.api_client.get_project_links()
+        if project_links is None:
+            return None
+        return len(project_links), project_links
+
+    def analyze(self) -> tuple[RESULT, dict]:
+        """Check whether the package contains one link.
```

Comment:
Hmmm I believe this heuristic is for checking whether a PyPI package has no project link. I think this doc string need to be updated.

Reply:
Fixed this in 71f87ac0e2f6c6fed616a773e43a40bb642e990a